lib/routes/blizzard/news-cn.ts

```
@@ -0,0 +1,133 @@
+                description: item.find('.content-intro').text(),
+                pubDate: parseDate(item.find('.content-date').text()),
+                image: item.find('.item-pic').attr('src'),
+            };
+        }),
+    hs: ($) =>
+        $('.article-container>a').toArray().map((item) => {
+            item = $(item);
+            return {
+                title: item.find('.title').text(),
+                link: item.attr('href'),
+                description: item.find('.desc').text(),
+                pubDate: parseDate(item.find('.date').attr('data-time')),
+                image: item.find('.article-img img').attr('src'),
+            };
+        }),
+    wow: ($) => $('.Pane-list>a').toArray().map((item) => {
+        item = $(item);
+        return {
+            title: item.find('.list-title').text(),
+            link: item.attr('href'),
+            description: item.find('.list-desc').text(),
+            pubDate: parseDate(item.find('.list-time').attr('data-time')),
+            image: item.find('.img-box img').attr('src'),
+        };
+    }),
+};
+
+// 详情页解析逻辑
+const detailParsers = {
+    ow: ($) => $('.detail-content-box').first().html(),
+    hs: ($) => $('.article').first().html(),
+    wow: ($) => $('.detail').first().html(),
+};
+
+function getList(category, $) {
+    return parsers[category] ? parsers[category]($) : [];
+}
+
+async function fetchDetail(item, category) {
+    return await cache.tryGet(item.link, async () => {
+        const response = await ofetch(item.link);
+        const $ = load(response);
+
+        const parseDetail = detailParsers[category];
+        item.description = parseDetail($);
+        return item;
+    });
+}
+
+async function handler(ctx) {
+    const category = ctx.req.param('category') || 'ow';
+    if (!categoryNames[category]) {
+        return {
+            title: '错误的类别',
+            description: '您请求的类别不存在。',
+            item: [],
+        };
```

Throw an exception instead of returning feed data.